src/renderer/components/wallet/Settings.tsx

```diff
@@ -64,111 +45,137 @@ const Settings: React.FC<Props> = (props: Props): JSX.Element => {
         userAccounts,
         O.map((accounts) => (
           <Col key={'accounts'} sm={{ span: 24 }} md={{ span: 12 }}>
-            <StyledSubtitle>{intl.formatMessage({ id: 'setting.account.management' })}</StyledSubtitle>
-            <StyledAccountCard>
+            <Styled.Subtitle>{intl.formatMessage({ id: 'setting.account.management' })}</Styled.Subtitle>
+            <Styled.AccountCard>
               <List
                 dataSource={accounts}
                 renderItem={(item, i: number) => (
-                  <StyledListItem key={i}>
-                    <StyledChainName>{item.chainName}</StyledChainName>
+                  <Styled.ListItem key={i}>
+                    <Styled.ChainName>{item.chainName}</Styled.ChainName>
                     {item.accounts.map((acc, j) => (
-                      <StyledChainContent key={j}>
-                        <StyledAccountPlaceholder>{acc.name}</StyledAccountPlaceholder>
-                        <StyledAccountContent>
-                          <StyledAccountAddress>{acc.address}</StyledAccountAddress>
+                      <Styled.ChainContent key={j}>
+                        <Styled.AccountPlaceholder>{acc.name}</Styled.AccountPlaceholder>
+                        <Styled.AccountContent>
+                          <Styled.AccountAddress>{acc.address}</Styled.AccountAddress>
                           {acc.type === 'external' && (
                             <Button type="link" danger>
                               <StopOutlined />
                             </Button>
                           )}
-                        </StyledAccountContent>
-                      </StyledChainContent>
+                        </Styled.AccountContent>
+                      </Styled.ChainContent>
                     ))}
-                  </StyledListItem>
+                  </Styled.ListItem>
                 )}
               />
-            </StyledAccountCard>
+            </Styled.AccountCard>
           </Col>
         )),
         O.getOrElse(() => <></>)
       ),
     [intl, userAccounts]
   )
 
+  const changeNetworkHandler: MenuProps['onClick'] = useCallback(
+    (param) => {
```

Comment:
here `param` has type any. why would not we type it. ?

Reply:
I could not figure out how to type it - `antd` is still not my friend here. Maybe you have an idea ...?

Reply:
oh gosh. nope. they have [MenuInfo interface](https://github.com/react-component/menu/blob/63571e1b845e12ffc1a46c1424f64a9ce3bc8a20/src/interface.ts#L5) but it is not exported anywhere. also `Menu` component is not generic so we can not pass a type-argument there :(